src/components/MyClosetPage/MyClosetPage.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components/macro'
+import Card from '../Card/Card'
+import Filter from '../Filter/Filter'
+
+export default function MyClosetPage({
+  userInput,
+  setUserInput,
+  cards,
+  setCards,
+}) {
+  return (
+    <MyClosetPageLayout>
+      <Filter userInput={userInput} setUserInput={setUserInput} />
+      <ResultContainer>
+        {cards
+          .filter(card =>
+            card.name.toLowerCase().includes(userInput.toLowerCase())
+          )
+          .map(
+            ({ name, image, id, store, price, date, clothingType, images }) => (
+              <Card
+                key={id}
+                image={image}
+                name={name}
+                store={store}
+                price={price}
+                date={date}
+                clothingType={clothingType}
+                images={images}
+              />
+            )
+          )}
+      </ResultContainer>
+    </MyClosetPageLayout>
+  )
+}
+const MyClosetPageLayout = styled.div`
```

Comment:
Suggestion:
If I have understood your code correctly, you could use `main` instead of `div` for more semantic HTML